ggshield/verticals/secret/secret_scan_collection.py

```diff
@@ -70,25 +135,13 @@ class Result:
     filemode: Filemode
     path: Path
     url: str
-    policy_breaks: List[PolicyBreak]
-    ignored_policy_breaks_count_by_reason: Counter[str]
+    policy_breaks: List[Secret]
+    ignored_policy_breaks_count_by_kind: Counter[IgnoreKind]
```

Comment:
Since we are already renaming this field, I think it would make sense to name it `ignored_secrets_count_by_kind`, but only if we rename `policy_breaks` to `secrets`. That would actually not be that invasive: it's only used in 11 places in the code (and 31 in tests).

Reply:
Added rename commit.